src/main/java/net/mcreator/ui/dialogs/wysiwyg/OutputSlotDialog.java

```diff
@@ -59,59 +60,57 @@ public OutputSlotDialog(WYSIWYGEditor editor, @Nullable OutputSlot slot) {
+		options.add(PanelUtils.join(FlowLayout.LEFT, L10N.label("dialog.slot.slot_id"), slotID));
 
-		JCheckBox disableStackInteraction = new JCheckBox(
-				"Disable player interaction with this slot (can't take item from it or insert in it)");
+		JCheckBox disableStackInteraction = L10N.checkbox("dialog.slot.disable_player_interaction");
 		options.add(PanelUtils.join(FlowLayout.LEFT, disableStackInteraction));
 
-		JCheckBox dropItemsWhenNotBound = new JCheckBox(
-				"Drop items when GUI not bound to any external inventory is closed");
+		JCheckBox dropItemsWhenNotBound = L10N.checkbox("dialog.slot.drop_item_when_gui_closed");
 		options.add(PanelUtils.join(FlowLayout.LEFT, dropItemsWhenNotBound));
 
 		dropItemsWhenNotBound.setSelected(true);
 
 		final JColor color = new JColor(editor.mcreator);
 		options.add(PanelUtils
-				.join(FlowLayout.LEFT, new JLabel("<html>Custom color<br><small>Set only if you want custom color: "),
+				.join(FlowLayout.LEFT, L10N.label("dialog.slot.custom_color"),
 						color));
 
 		ProcedureSelector eh = new ProcedureSelector(IHelpContext.NONE.withEntry("gui/when_slot_changed"),
-				editor.mcreator, "When slot contents change", ProcedureSelector.Side.BOTH, false,
+				editor.mcreator, L10N.t("dialog.slot.event_slot_content_changes"), ProcedureSelector.Side.BOTH, false,
 				Dependency.fromString("x:number/y:number/z:number/world:world/entity:entity/guistate:map"));
 		eh.refreshList();
 
 		ProcedureSelector eh2 = new ProcedureSelector(IHelpContext.NONE.withEntry("gui/when_slot_item_taken"),
-				editor.mcreator, "When item taken from slot", ProcedureSelector.Side.BOTH, false,
+				editor.mcreator, L10N.t("dialog.slot.event_item_taken_from_slot"), ProcedureSelector.Side.BOTH, false,
 				Dependency.fromString("x:number/y:number/z:number/world:world/entity:entity/guistate:map"));
 		eh2.refreshList();
 
 		ProcedureSelector eh3 = new ProcedureSelector(IHelpContext.NONE.withEntry("gui/when_transferred_from_slot"),
-				editor.mcreator, "When transferred from slot (shift-click)", ProcedureSelector.Side.BOTH, false,
+				editor.mcreator, L10N.t("dialog.slot.event_transfered_from_slot"), ProcedureSelector.Side.BOTH, false,
 				Dependency
 						.fromString("x:number/y:number/z:number/world:world/entity:entity/guistate:map/amount:number"));
 		eh3.refreshList();
 
 		add("Center", new JScrollPane(PanelUtils.centerInPanel(PanelUtils.gridElements(1, 3, 5, 5, eh, eh2, eh3))));
 
 		add("North", PanelUtils.join(FlowLayout.LEFT, options));
-		setTitle("Output slot editor");
-		JButton ok = new JButton("Save slot");
+		setTitle(L10N.t("dialog.slot.output_editor_title"));
+		JButton ok = L10N.button("dialog.slot.save_slot");
 		JButton cancel = new JButton(UIManager.getString("OptionPane.cancelButtonText"));
 		add("South", PanelUtils.join(ok, cancel));
 
 		getRootPane().setDefaultButton(ok);
 
 		if (slot != null) {
-			ok.setText("Save changes");

```

Comment:
Save changes is quite general so I think this one is safe to move to common.save_changes

Reply:
Done!